airbyte-integrations/connectors/destination-snowflake/src/test-integration/resources/sqlgenerator/reservedkeywords_expectedrecords_final.jsonl

```diff
@@ -0,0 +1 @@
+{"_airbyte_raw_id":"b2e0efc4-38a8-47ba-970c-8103f09f08d5","_airbyte_extracted_at":"2023-01-01T00:00:00.000000000Z","_airbyte_meta":{"errors":[]}, "_CURRENT_DATE": "foo"}
```

Comment:
comments! :D 
```suggestion
// Note that we prefix `current_date` with an underscore (to `_CURRENT_DATE`). This is because even if you quote `CURRENT_DATE`, snowflake rejects it as a column name.
{"_airbyte_raw_id":"b2e0efc4-38a8-47ba-970c-8103f09f08d5","_airbyte_extracted_at":"2023-01-01T00:00:00.000000000Z","_airbyte_meta":{"errors":[]}, "_CURRENT_DATE": "foo"}
```